src/main/java/github/scarsz/discordsrv/objects/threads/ServerWatchdog.java

```diff
@@ -66,8 +68,22 @@ public void run() {
                             .replace("%guildowner%", DiscordSRV.getPlugin().getMainGuild().getOwner().getAsMention())
                     );
 
-                    for (int i = 0; i < DiscordSRV.config().getInt("ServerWatchdogMessageCount"); i++) {
-                        DiscordUtil.sendMessage(DiscordSRV.getPlugin().getMainTextChannel(), message);
+                    String channelName = DiscordSRV.getPlugin().getMainTextChannel().getName();
+                    Integer count = DiscordSRV.config().getInt("ServerWatchdogMessageCount");
+
+                    WatchdogMessageEvent event = DiscordSRV.api.callEvent(new WatchdogMessageEvent(channelName, message, count, false));
+                    if (event.isCancelled()) {
+                        DiscordSRV.debug("WatchdogMessageEvent was cancelled, message send aborted");
+                        return;
+                    }
+                    count = event.getCount();         // update count from event in case any listeners modified it
+                    channelName = event.getChannel(); // update channel from event in case any listeners modified it
+                    message = event.getMessage();     // update message from event in case any listeners modified it
```

Comment:
this would be much better suited as one single comment above the three setters, the action don't really change between the lines besides the variables